Build failure: mpv-unwrapped

### Nixpkgs version

- Unstable (26.11)

### Steps to reproduce

-

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/336948122

### Relevant log output

```console
[260/261] Linking target mpv
FAILED: [code=1] mpv 
clang  -o mpv osdep/mac/swift.o libmpv.2.dylib.p/audio_aframe.c.o libmpv.2.dylib.p/audio_chmap.c.o libmpv.2.dylib.p/audio_chmap_avchannel.c.o libmpv.2.dylib.p/audio_chmap_sel.c.o libmpv.2.dylib.p/audio_decode_ad_lavc.c.o libmpv.2.dylib.p/audio_decode_ad_spdif.c.o libmpv.2.dylib.p/audio_filter_af_drop.c.o libmpv.2.dylib.p/audio_filter_af_format.c.o libmpv.2.dylib.p/audio_filter_af_lavcac3enc.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo2.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo2_internals.c.o libmpv.2.dylib.p/audio_fmt-conversion.c.o libmpv.2.dylib.p/audio_format.c.o libmpv.2.dylib.p/audio_out_ao.c.o libmpv.2.dylib.p/audio_out_ao_lavc.c.o libmpv.2.dylib.p/audio_out_ao_null.c.o libmpv.2.dylib.p/audio_out_ao_pcm.c.o libmpv.2.dylib.p/audio_out_buffer.c.o libmpv.2.dylib.p/common_av_common.c.o libmpv.2.dylib.p/common_av_log.c.o libmpv.2.dylib.p/common_codecs.c.o libmpv.2.dylib.p/common_common.c.o libmpv.2.dylib.p/common_encode_lavc.c.o libmpv.2.dylib.p/common_msg.c.o libmpv.2.dylib.p/common_playlist.c.o libmpv.2.dylib.p/common_recorder.c.o libmpv.2.dylib.p/common_stats.c.o libmpv.2.dylib.p/common_tags.c.o libmpv.2.dylib.p/common_version.c.o libmpv.2.dylib.p/demux_codec_tags.c.o libmpv.2.dylib.p/demux_cue.c.o libmpv.2.dylib.p/demux_cache.c.o libmpv.2.dylib.p/demux_demux.c.o libmpv.2.dylib.p/demux_demux_cue.c.o libmpv.2.dylib.p/demux_demux_disc.c.o libmpv.2.dylib.p/demux_demux_edl.c.o libmpv.2.dylib.p/demux_demux_lavf.c.o libmpv.2.dylib.p/demux_demux_mf.c.o libmpv.2.dylib.p/demux_demux_mkv.c.o libmpv.2.dylib.p/demux_demux_mkv_timeline.c.o libmpv.2.dylib.p/demux_demux_mpv.c.o libmpv.2.dylib.p/demux_demux_null.c.o libmpv.2.dylib.p/demux_demux_playlist.c.o libmpv.2.dylib.p/demux_demux_raw.c.o libmpv.2.dylib.p/demux_demux_timeline.c.o libmpv.2.dylib.p/demux_ebml.c.o libmpv.2.dylib.p/demux_packet.c.o libmpv.2.dylib.p/demux_packet_pool.c.o libmpv.2.dylib.p/demux_timeline.c.o libmpv.2.dylib.p/filters_f_async_queue.c.o libmpv.2.dylib.p/filters_f_autoconvert.c.o libmpv.2.dylib.p/filters_f_auto_filters.c.o libmpv.2.dylib.p/filters_f_decoder_wrapper.c.o libmpv.2.dylib.p/filters_f_demux_in.c.o libmpv.2.dylib.p/filters_f_hwtransfer.c.o libmpv.2.dylib.p/filters_f_lavfi.c.o libmpv.2.dylib.p/filters_f_output_chain.c.o libmpv.2.dylib.p/filters_f_swresample.c.o libmpv.2.dylib.p/filters_f_swscale.c.o libmpv.2.dylib.p/filters_f_utils.c.o libmpv.2.dylib.p/filters_filter.c.o libmpv.2.dylib.p/filters_frame.c.o libmpv.2.dylib.p/filters_user_filters.c.o libmpv.2.dylib.p/input_cmd.c.o libmpv.2.dylib.p/input_event.c.o libmpv.2.dylib.p/input_input.c.o libmpv.2.dylib.p/input_ipc.c.o libmpv.2.dylib.p/input_keycodes.c.o libmpv.2.dylib.p/misc_bstr.c.o libmpv.2.dylib.p/misc_charset_conv.c.o libmpv.2.dylib.p/misc_codepoint_width.c.o libmpv.2.dylib.p/misc_dispatch.c.o libmpv.2.dylib.p/misc_io_utils.c.o libmpv.2.dylib.p/misc_json.c.o libmpv.2.dylib.p/misc_language.c.o libmpv.2.dylib.p/misc_natural_sort.c.o libmpv.2.dylib.p/misc_node.c.o libmpv.2.dylib.p/misc_path_utils.c.o libmpv.2.dylib.p/misc_random.c.o libmpv.2.dylib.p/misc_rendezvous.c.o libmpv.2.dylib.p/misc_thread_pool.c.o libmpv.2.dylib.p/misc_thread_tools.c.o libmpv.2.dylib.p/options_m_config_core.c.o libmpv.2.dylib.p/options_m_config_frontend.c.o libmpv.2.dylib.p/options_m_option.c.o libmpv.2.dylib.p/options_m_property.c.o libmpv.2.dylib.p/options_options.c.o libmpv.2.dylib.p/options_parse_commandline.c.o libmpv.2.dylib.p/options_parse_configfile.c.o libmpv.2.dylib.p/options_path.c.o libmpv.2.dylib.p/player_audio.c.o libmpv.2.dylib.p/player_client.c.o libmpv.2.dylib.p/player_command.c.o libmpv.2.dylib.p/player_configfiles.c.o libmpv.2.dylib.p/player_external_files.c.o libmpv.2.dylib.p/player_loadfile.c.o libmpv.2.dylib.p/player_main.c.o libmpv.2.dylib.p/player_misc.c.o libmpv.2.dylib.p/player_osd.c.o libmpv.2.dylib.p/player_playloop.c.o libmpv.2.dylib.p/player_screenshot.c.o libmpv.2.dylib.p/player_scripting.c.o libmpv.2.dylib.p/player_sub.c.o libmpv.2.dylib.p/player_video.c.o libmpv.2.dylib.p/player_clipboard_clipboard.c.o libmpv.2.dylib.p/player_clipboard_clipboard-vo.c.o libmpv.2.dylib.p/stream_cookies.c.o libmpv.2.dylib.p/stream_stream.c.o libmpv.2.dylib.p/stream_stream_avdevice.c.o libmpv.2.dylib.p/stream_stream_cb.c.o libmpv.2.dylib.p/stream_stream_concat.c.o libmpv.2.dylib.p/stream_stream_edl.c.o libmpv.2.dylib.p/stream_stream_file.c.o libmpv.2.dylib.p/stream_stream_lavf.c.o libmpv.2.dylib.p/stream_stream_memory.c.o libmpv.2.dylib.p/stream_stream_mf.c.o libmpv.2.dylib.p/stream_stream_mpv.c.o libmpv.2.dylib.p/stream_stream_null.c.o libmpv.2.dylib.p/stream_stream_slice.c.o libmpv.2.dylib.p/sub_ass_mp.c.o libmpv.2.dylib.p/sub_dec_sub.c.o libmpv.2.dylib.p/sub_draw_bmp.c.o libmpv.2.dylib.p/sub_filter_sdh.c.o libmpv.2.dylib.p/sub_img_convert.c.o libmpv.2.dylib.p/sub_lavc_conv.c.o libmpv.2.dylib.p/sub_osd.c.o libmpv.2.dylib.p/sub_osd_libass.c.o libmpv.2.dylib.p/sub_sd_ass.c.o libmpv.2.dylib.p/sub_sd_lavc.c.o libmpv.2.dylib.p/video_csputils.c.o libmpv.2.dylib.p/video_decode_vd_lavc.c.o libmpv.2.dylib.p/video_filter_refqueue.c.o libmpv.2.dylib.p/video_filter_vf_format.c.o libmpv.2.dylib.p/video_filter_vf_sub.c.o libmpv.2.dylib.p/video_fmt-conversion.c.o libmpv.2.dylib.p/video_hwdec.c.o libmpv.2.dylib.p/video_image_loader.c.o libmpv.2.dylib.p/video_image_writer.c.o libmpv.2.dylib.p/video_img_format.c.o libmpv.2.dylib.p/video_mp_image.c.o libmpv.2.dylib.p/video_mp_image_pool.c.o libmpv.2.dylib.p/video_out_aspect.c.o libmpv.2.dylib.p/video_out_bitmap_packer.c.o libmpv.2.dylib.p/video_out_dither.c.o libmpv.2.dylib.p/video_out_dr_helper.c.o libmpv.2.dylib.p/video_out_filter_kernels.c.o libmpv.2.dylib.p/video_out_gpu_context.c.o libmpv.2.dylib.p/video_out_gpu_error_diffusion.c.o libmpv.2.dylib.p/video_out_gpu_hwdec.c.o libmpv.2.dylib.p/video_out_gpu_lcms.c.o libmpv.2.dylib.p/video_out_gpu_libmpv_gpu.c.o libmpv.2.dylib.p/video_out_gpu_osd.c.o libmpv.2.dylib.p/video_out_gpu_ra.c.o libmpv.2.dylib.p/video_out_gpu_shader_cache.c.o libmpv.2.dylib.p/video_out_gpu_spirv.c.o libmpv.2.dylib.p/video_out_gpu_user_shaders.c.o libmpv.2.dylib.p/video_out_gpu_utils.c.o libmpv.2.dylib.p/video_out_gpu_video.c.o libmpv.2.dylib.p/video_out_gpu_video_shaders.c.o libmpv.2.dylib.p/video_out_libmpv_sw.c.o libmpv.2.dylib.p/video_out_vo.c.o libmpv.2.dylib.p/video_out_vo_gpu.c.o libmpv.2.dylib.p/video_out_vo_image.c.o libmpv.2.dylib.p/video_out_vo_lavc.c.o libmpv.2.dylib.p/video_out_vo_libmpv.c.o libmpv.2.dylib.p/video_out_vo_null.c.o libmpv.2.dylib.p/video_out_vo_tct.c.o libmpv.2.dylib.p/video_out_vo_kitty.c.o libmpv.2.dylib.p/video_out_win_state.c.o libmpv.2.dylib.p/video_repack.c.o libmpv.2.dylib.p/video_sws_utils.c.o libmpv.2.dylib.p/video_out_placebo_ra_pl.c.o libmpv.2.dylib.p/video_out_placebo_utils.c.o libmpv.2.dylib.p/video_out_vo_gpu_next.c.o libmpv.2.dylib.p/video_out_gpu_next_context.c.o libmpv.2.dylib.p/osdep_io.c.o libmpv.2.dylib.p/osdep_subprocess.c.o libmpv.2.dylib.p/osdep_timer.c.o libmpv.2.dylib.p/ta_ta.c.o libmpv.2.dylib.p/ta_ta_talloc.c.o libmpv.2.dylib.p/ta_ta_utils.c.o libmpv.2.dylib.p/osdep_threads-posix.c.o libmpv.2.dylib.p/osdep_language-mac.c.o libmpv.2.dylib.p/osdep_utils-mac.c.o libmpv.2.dylib.p/osdep_mac_app_bridge.m.o libmpv.2.dylib.p/player_clipboard_clipboard-mac.m.o libmpv.2.dylib.p/osdep_subprocess-posix.c.o libmpv.2.dylib.p/input_ipc-unix.c.o libmpv.2.dylib.p/osdep_poll_wrapper.c.o libmpv.2.dylib.p/osdep_terminal-unix.c.o libmpv.2.dylib.p/sub_filter_regex.c.o libmpv.2.dylib.p/osdep_als-voctrl.c.o libmpv.2.dylib.p/osdep_path-darwin.c.o libmpv.2.dylib.p/osdep_path-mac.m.o libmpv.2.dylib.p/osdep_timer-darwin.c.o libmpv.2.dylib.p/stream_stream_dvdnav.c.o libmpv.2.dylib.p/player_javascript.c.o libmpv.2.dylib.p/sub_filter_jsre.c.o libmpv.2.dylib.p/demux_demux_libarchive.c.o libmpv.2.dylib.p/stream_stream_libarchive.c.o libmpv.2.dylib.p/stream_stream_bluray.c.o libmpv.2.dylib.p/player_lua.c.o libmpv.2.dylib.p/audio_filter_af_rubberband.c.o libmpv.2.dylib.p/video_filter_vf_fingerprint.c.o libmpv.2.dylib.p/video_zimg.c.o libmpv.2.dylib.p/audio_out_ao_avfoundation.m.o libmpv.2.dylib.p/audio_out_ao_coreaudio.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_exclusive.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_properties.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_chmap.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_utils.c.o libmpv.2.dylib.p/audio_out_ao_openal.c.o libmpv.2.dylib.p/video_out_vo_caca.c.o libmpv.2.dylib.p/video_out_opengl_common.c.o libmpv.2.dylib.p/video_out_opengl_context.c.o libmpv.2.dylib.p/video_out_opengl_formats.c.o libmpv.2.dylib.p/video_out_opengl_libmpv_gl.c.o libmpv.2.dylib.p/video_out_opengl_ra_gl.c.o libmpv.2.dylib.p/video_out_opengl_utils.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vulkan.c.o libmpv.2.dylib.p/video_out_vulkan_context.c.o libmpv.2.dylib.p/video_out_vulkan_utils.c.o libmpv.2.dylib.p/video_filter_vf_gpu_vulkan.c.o libmpv.2.dylib.p/video_out_vulkan_context_display.c.o libmpv.2.dylib.p/video_filter_vf_gpu.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vt.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_mac_gl.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vt_pl.m.o libmpv.2.dylib.p/video_out_vulkan_context_mac.m.o mpv.p/osdep_main-fn-mac.c.o -L/usr/lib -L/usr/local/lib -L/nix/store/if4rm04h2fpq5kw7zyfkyh0gf6cq6faq-swift-5.10.1-lib/lib/swift/macosx -L/usr/lib/swift -Wl,-dead_strip_dylibs -Wl,-headerpad_max_install_names -Wl,-undefined,dynamic_lookup -rdynamic -isysroot /nix/store/mxzgf8zlr2mbxrqp1ami2ixqsqpskv0w-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -Xlinker -rpath -Xlinker /nix/store/if4rm04h2fpq5kw7zyfkyh0gf6cq6faq-swift-5.10.1-lib/lib/swift/macosx -rdynamic -Xlinker -add_ast_path -Xlinker /nix/var/nix/builds/nix-12544-2708964383/source/build/osdep/mac/swift.swiftmodule -Xlinker -rpath -Xlinker /usr/lib/swift -Wl,-rpath,/nix/store/jfl0bsshv9cg742ig6x90vwn64jak8aq-lua-5.2.4/lib /nix/store/p9vprkv7dkkmyvaws4414gggmpxis76n-libass-0.17.4/lib/libass.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavcodec.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavfilter.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavformat.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavutil.dylib /nix/store/iyb2vjwf0yw99vy296afmb3gribbf502-libplacebo-7.360.1/lib/libplacebo.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libswresample.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libswscale.dylib -framework Cocoa -framework IOKit -framework QuartzCore /nix/store/0mjqhskci9ja8z9y41mha96nb1mlsi68-libdvdnav-6.1.1/lib/libdvdnav.dylib -lpthread -liconv /nix/store/cawaxcrrcxx3mipl64760k0p75p3n48q-mujs-1.3.6/lib/libmujs.dylib /nix/store/4frg9qwhvaziyk6m4f1jqmchp204501q-lcms2-2.19.1/lib/liblcms2.dylib /nix/store/92ay55bcwsj41jdk1ng0fvda7cbvd9k7-libarchive-3.8.8-lib/lib/libarchive.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavdevice.dylib /nix/store/pp0qn8mxm8mq949wy87km551zrbna0qq-libbluray-1.4.1/lib/libbluray.dylib -lm /nix/store/jfl0bsshv9cg742ig6x90vwn64jak8aq-lua-5.2.4/lib/liblua.dylib /nix/store/4w6nxsq2csf3aj6qggy0rr5fn224ykgx-rubberband-4.0.0/lib/librubberband.dylib -framework Accelerate /nix/store/bnam1gnifvm2s1dab3d23nqfymc8r83c-uchardet-0.0.8/lib/libuchardet.dylib /nix/store/fr71skraxgk85biax0zxrln93zi5bh16-zimg-3.0.6/lib/libzimg.dylib /nix/store/r1rf9w7c2pihgk4snl9xvyr90m9pp3ij-zlib-1.3.2-static/lib/libz.a -framework CoreMedia -framework AVFoundation -framework CoreFoundation -framework CoreAudio -framework AudioUnit -framework AudioToolbox /nix/store/cyxlngr35560g9b4gympq10cs4h6mnp0-openal-soft-1.24.3/lib/libopenal.dylib /nix/store/qxpi14i3ba0sm0m7bb0l8dzizjgwx8fs-libcaca-0.99.beta20/lib/libcaca.dylib -framework OpenGL /nix/store/4xhqv41dryk2i8a98bh5jbqxfnh5n8c5-vulkan-loader-1.4.350.0/lib/libvulkan.dylib -framework CoreVideo
/nix/store/pnd38c1w69ajjfmpsajbz7i72jd2i62v-cctools-binutils-darwin-wrapper-1010.6/bin/ld: line 292: 29875 Trace/BPT trap: 5          /nix/store/zhb8xrwn0mp0xk3argfbh8w73c4cipqs-cctools-binutils-darwin-1010.6/bin/ld "@$responseFile"
clang: error: linker command failed with exit code 133 (use -v to see invocation)
[261/261] Linking target libmpv.2.dylib
FAILED: [code=1] libmpv.2.dylib 
clang  -o libmpv.2.dylib osdep/mac/swift.o libmpv.2.dylib.p/audio_aframe.c.o libmpv.2.dylib.p/audio_chmap.c.o libmpv.2.dylib.p/audio_chmap_avchannel.c.o libmpv.2.dylib.p/audio_chmap_sel.c.o libmpv.2.dylib.p/audio_decode_ad_lavc.c.o libmpv.2.dylib.p/audio_decode_ad_spdif.c.o libmpv.2.dylib.p/audio_filter_af_drop.c.o libmpv.2.dylib.p/audio_filter_af_format.c.o libmpv.2.dylib.p/audio_filter_af_lavcac3enc.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo2.c.o libmpv.2.dylib.p/audio_filter_af_scaletempo2_internals.c.o libmpv.2.dylib.p/audio_fmt-conversion.c.o libmpv.2.dylib.p/audio_format.c.o libmpv.2.dylib.p/audio_out_ao.c.o libmpv.2.dylib.p/audio_out_ao_lavc.c.o libmpv.2.dylib.p/audio_out_ao_null.c.o libmpv.2.dylib.p/audio_out_ao_pcm.c.o libmpv.2.dylib.p/audio_out_buffer.c.o libmpv.2.dylib.p/common_av_common.c.o libmpv.2.dylib.p/common_av_log.c.o libmpv.2.dylib.p/common_codecs.c.o libmpv.2.dylib.p/common_common.c.o libmpv.2.dylib.p/common_encode_lavc.c.o libmpv.2.dylib.p/common_msg.c.o libmpv.2.dylib.p/common_playlist.c.o libmpv.2.dylib.p/common_recorder.c.o libmpv.2.dylib.p/common_stats.c.o libmpv.2.dylib.p/common_tags.c.o libmpv.2.dylib.p/common_version.c.o libmpv.2.dylib.p/demux_codec_tags.c.o libmpv.2.dylib.p/demux_cue.c.o libmpv.2.dylib.p/demux_cache.c.o libmpv.2.dylib.p/demux_demux.c.o libmpv.2.dylib.p/demux_demux_cue.c.o libmpv.2.dylib.p/demux_demux_disc.c.o libmpv.2.dylib.p/demux_demux_edl.c.o libmpv.2.dylib.p/demux_demux_lavf.c.o libmpv.2.dylib.p/demux_demux_mf.c.o libmpv.2.dylib.p/demux_demux_mkv.c.o libmpv.2.dylib.p/demux_demux_mkv_timeline.c.o libmpv.2.dylib.p/demux_demux_mpv.c.o libmpv.2.dylib.p/demux_demux_null.c.o libmpv.2.dylib.p/demux_demux_playlist.c.o libmpv.2.dylib.p/demux_demux_raw.c.o libmpv.2.dylib.p/demux_demux_timeline.c.o libmpv.2.dylib.p/demux_ebml.c.o libmpv.2.dylib.p/demux_packet.c.o libmpv.2.dylib.p/demux_packet_pool.c.o libmpv.2.dylib.p/demux_timeline.c.o libmpv.2.dylib.p/filters_f_async_queue.c.o libmpv.2.dylib.p/filters_f_autoconvert.c.o libmpv.2.dylib.p/filters_f_auto_filters.c.o libmpv.2.dylib.p/filters_f_decoder_wrapper.c.o libmpv.2.dylib.p/filters_f_demux_in.c.o libmpv.2.dylib.p/filters_f_hwtransfer.c.o libmpv.2.dylib.p/filters_f_lavfi.c.o libmpv.2.dylib.p/filters_f_output_chain.c.o libmpv.2.dylib.p/filters_f_swresample.c.o libmpv.2.dylib.p/filters_f_swscale.c.o libmpv.2.dylib.p/filters_f_utils.c.o libmpv.2.dylib.p/filters_filter.c.o libmpv.2.dylib.p/filters_frame.c.o libmpv.2.dylib.p/filters_user_filters.c.o libmpv.2.dylib.p/input_cmd.c.o libmpv.2.dylib.p/input_event.c.o libmpv.2.dylib.p/input_input.c.o libmpv.2.dylib.p/input_ipc.c.o libmpv.2.dylib.p/input_keycodes.c.o libmpv.2.dylib.p/misc_bstr.c.o libmpv.2.dylib.p/misc_charset_conv.c.o libmpv.2.dylib.p/misc_codepoint_width.c.o libmpv.2.dylib.p/misc_dispatch.c.o libmpv.2.dylib.p/misc_io_utils.c.o libmpv.2.dylib.p/misc_json.c.o libmpv.2.dylib.p/misc_language.c.o libmpv.2.dylib.p/misc_natural_sort.c.o libmpv.2.dylib.p/misc_node.c.o libmpv.2.dylib.p/misc_path_utils.c.o libmpv.2.dylib.p/misc_random.c.o libmpv.2.dylib.p/misc_rendezvous.c.o libmpv.2.dylib.p/misc_thread_pool.c.o libmpv.2.dylib.p/misc_thread_tools.c.o libmpv.2.dylib.p/options_m_config_core.c.o libmpv.2.dylib.p/options_m_config_frontend.c.o libmpv.2.dylib.p/options_m_option.c.o libmpv.2.dylib.p/options_m_property.c.o libmpv.2.dylib.p/options_options.c.o libmpv.2.dylib.p/options_parse_commandline.c.o libmpv.2.dylib.p/options_parse_configfile.c.o libmpv.2.dylib.p/options_path.c.o libmpv.2.dylib.p/player_audio.c.o libmpv.2.dylib.p/player_client.c.o libmpv.2.dylib.p/player_command.c.o libmpv.2.dylib.p/player_configfiles.c.o libmpv.2.dylib.p/player_external_files.c.o libmpv.2.dylib.p/player_loadfile.c.o libmpv.2.dylib.p/player_main.c.o libmpv.2.dylib.p/player_misc.c.o libmpv.2.dylib.p/player_osd.c.o libmpv.2.dylib.p/player_playloop.c.o libmpv.2.dylib.p/player_screenshot.c.o libmpv.2.dylib.p/player_scripting.c.o libmpv.2.dylib.p/player_sub.c.o libmpv.2.dylib.p/player_video.c.o libmpv.2.dylib.p/player_clipboard_clipboard.c.o libmpv.2.dylib.p/player_clipboard_clipboard-vo.c.o libmpv.2.dylib.p/stream_cookies.c.o libmpv.2.dylib.p/stream_stream.c.o libmpv.2.dylib.p/stream_stream_avdevice.c.o libmpv.2.dylib.p/stream_stream_cb.c.o libmpv.2.dylib.p/stream_stream_concat.c.o libmpv.2.dylib.p/stream_stream_edl.c.o libmpv.2.dylib.p/stream_stream_file.c.o libmpv.2.dylib.p/stream_stream_lavf.c.o libmpv.2.dylib.p/stream_stream_memory.c.o libmpv.2.dylib.p/stream_stream_mf.c.o libmpv.2.dylib.p/stream_stream_mpv.c.o libmpv.2.dylib.p/stream_stream_null.c.o libmpv.2.dylib.p/stream_stream_slice.c.o libmpv.2.dylib.p/sub_ass_mp.c.o libmpv.2.dylib.p/sub_dec_sub.c.o libmpv.2.dylib.p/sub_draw_bmp.c.o libmpv.2.dylib.p/sub_filter_sdh.c.o libmpv.2.dylib.p/sub_img_convert.c.o libmpv.2.dylib.p/sub_lavc_conv.c.o libmpv.2.dylib.p/sub_osd.c.o libmpv.2.dylib.p/sub_osd_libass.c.o libmpv.2.dylib.p/sub_sd_ass.c.o libmpv.2.dylib.p/sub_sd_lavc.c.o libmpv.2.dylib.p/video_csputils.c.o libmpv.2.dylib.p/video_decode_vd_lavc.c.o libmpv.2.dylib.p/video_filter_refqueue.c.o libmpv.2.dylib.p/video_filter_vf_format.c.o libmpv.2.dylib.p/video_filter_vf_sub.c.o libmpv.2.dylib.p/video_fmt-conversion.c.o libmpv.2.dylib.p/video_hwdec.c.o libmpv.2.dylib.p/video_image_loader.c.o libmpv.2.dylib.p/video_image_writer.c.o libmpv.2.dylib.p/video_img_format.c.o libmpv.2.dylib.p/video_mp_image.c.o libmpv.2.dylib.p/video_mp_image_pool.c.o libmpv.2.dylib.p/video_out_aspect.c.o libmpv.2.dylib.p/video_out_bitmap_packer.c.o libmpv.2.dylib.p/video_out_dither.c.o libmpv.2.dylib.p/video_out_dr_helper.c.o libmpv.2.dylib.p/video_out_filter_kernels.c.o libmpv.2.dylib.p/video_out_gpu_context.c.o libmpv.2.dylib.p/video_out_gpu_error_diffusion.c.o libmpv.2.dylib.p/video_out_gpu_hwdec.c.o libmpv.2.dylib.p/video_out_gpu_lcms.c.o libmpv.2.dylib.p/video_out_gpu_libmpv_gpu.c.o libmpv.2.dylib.p/video_out_gpu_osd.c.o libmpv.2.dylib.p/video_out_gpu_ra.c.o libmpv.2.dylib.p/video_out_gpu_shader_cache.c.o libmpv.2.dylib.p/video_out_gpu_spirv.c.o libmpv.2.dylib.p/video_out_gpu_user_shaders.c.o libmpv.2.dylib.p/video_out_gpu_utils.c.o libmpv.2.dylib.p/video_out_gpu_video.c.o libmpv.2.dylib.p/video_out_gpu_video_shaders.c.o libmpv.2.dylib.p/video_out_libmpv_sw.c.o libmpv.2.dylib.p/video_out_vo.c.o libmpv.2.dylib.p/video_out_vo_gpu.c.o libmpv.2.dylib.p/video_out_vo_image.c.o libmpv.2.dylib.p/video_out_vo_lavc.c.o libmpv.2.dylib.p/video_out_vo_libmpv.c.o libmpv.2.dylib.p/video_out_vo_null.c.o libmpv.2.dylib.p/video_out_vo_tct.c.o libmpv.2.dylib.p/video_out_vo_kitty.c.o libmpv.2.dylib.p/video_out_win_state.c.o libmpv.2.dylib.p/video_repack.c.o libmpv.2.dylib.p/video_sws_utils.c.o libmpv.2.dylib.p/video_out_placebo_ra_pl.c.o libmpv.2.dylib.p/video_out_placebo_utils.c.o libmpv.2.dylib.p/video_out_vo_gpu_next.c.o libmpv.2.dylib.p/video_out_gpu_next_context.c.o libmpv.2.dylib.p/osdep_io.c.o libmpv.2.dylib.p/osdep_subprocess.c.o libmpv.2.dylib.p/osdep_timer.c.o libmpv.2.dylib.p/ta_ta.c.o libmpv.2.dylib.p/ta_ta_talloc.c.o libmpv.2.dylib.p/ta_ta_utils.c.o libmpv.2.dylib.p/osdep_threads-posix.c.o libmpv.2.dylib.p/osdep_language-mac.c.o libmpv.2.dylib.p/osdep_utils-mac.c.o libmpv.2.dylib.p/osdep_mac_app_bridge.m.o libmpv.2.dylib.p/player_clipboard_clipboard-mac.m.o libmpv.2.dylib.p/osdep_subprocess-posix.c.o libmpv.2.dylib.p/input_ipc-unix.c.o libmpv.2.dylib.p/osdep_poll_wrapper.c.o libmpv.2.dylib.p/osdep_terminal-unix.c.o libmpv.2.dylib.p/sub_filter_regex.c.o libmpv.2.dylib.p/osdep_als-voctrl.c.o libmpv.2.dylib.p/osdep_path-darwin.c.o libmpv.2.dylib.p/osdep_path-mac.m.o libmpv.2.dylib.p/osdep_timer-darwin.c.o libmpv.2.dylib.p/stream_stream_dvdnav.c.o libmpv.2.dylib.p/player_javascript.c.o libmpv.2.dylib.p/sub_filter_jsre.c.o libmpv.2.dylib.p/demux_demux_libarchive.c.o libmpv.2.dylib.p/stream_stream_libarchive.c.o libmpv.2.dylib.p/stream_stream_bluray.c.o libmpv.2.dylib.p/player_lua.c.o libmpv.2.dylib.p/audio_filter_af_rubberband.c.o libmpv.2.dylib.p/video_filter_vf_fingerprint.c.o libmpv.2.dylib.p/video_zimg.c.o libmpv.2.dylib.p/audio_out_ao_avfoundation.m.o libmpv.2.dylib.p/audio_out_ao_coreaudio.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_exclusive.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_properties.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_chmap.c.o libmpv.2.dylib.p/audio_out_ao_coreaudio_utils.c.o libmpv.2.dylib.p/audio_out_ao_openal.c.o libmpv.2.dylib.p/video_out_vo_caca.c.o libmpv.2.dylib.p/video_out_opengl_common.c.o libmpv.2.dylib.p/video_out_opengl_context.c.o libmpv.2.dylib.p/video_out_opengl_formats.c.o libmpv.2.dylib.p/video_out_opengl_libmpv_gl.c.o libmpv.2.dylib.p/video_out_opengl_ra_gl.c.o libmpv.2.dylib.p/video_out_opengl_utils.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vulkan.c.o libmpv.2.dylib.p/video_out_vulkan_context.c.o libmpv.2.dylib.p/video_out_vulkan_utils.c.o libmpv.2.dylib.p/video_filter_vf_gpu_vulkan.c.o libmpv.2.dylib.p/video_out_vulkan_context_display.c.o libmpv.2.dylib.p/video_filter_vf_gpu.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vt.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_mac_gl.c.o libmpv.2.dylib.p/video_out_hwdec_hwdec_vt_pl.m.o libmpv.2.dylib.p/video_out_vulkan_context_mac.m.o -L/usr/lib -L/usr/local/lib -L/nix/store/if4rm04h2fpq5kw7zyfkyh0gf6cq6faq-swift-5.10.1-lib/lib/swift/macosx -L/usr/lib/swift -Wl,-dead_strip_dylibs -Wl,-headerpad_max_install_names -Wl,-undefined,dynamic_lookup -shared -install_name @rpath/libmpv.2.dylib -compatibility_version 2 -current_version 2 -rdynamic -isysroot /nix/store/mxzgf8zlr2mbxrqp1ami2ixqsqpskv0w-apple-sdk-14.4/Platforms/MacOSX.platform/Developer/SDKs/MacOSX14.4.sdk -Xlinker -rpath -Xlinker /nix/store/if4rm04h2fpq5kw7zyfkyh0gf6cq6faq-swift-5.10.1-lib/lib/swift/macosx -rdynamic -Xlinker -add_ast_path -Xlinker /nix/var/nix/builds/nix-12544-2708964383/source/build/osdep/mac/swift.swiftmodule -Xlinker -rpath -Xlinker /usr/lib/swift -Wl,-rpath,/nix/store/jfl0bsshv9cg742ig6x90vwn64jak8aq-lua-5.2.4/lib /nix/store/p9vprkv7dkkmyvaws4414gggmpxis76n-libass-0.17.4/lib/libass.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavcodec.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavfilter.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavformat.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavutil.dylib /nix/store/iyb2vjwf0yw99vy296afmb3gribbf502-libplacebo-7.360.1/lib/libplacebo.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libswresample.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libswscale.dylib -framework Cocoa -framework IOKit -framework QuartzCore /nix/store/0mjqhskci9ja8z9y41mha96nb1mlsi68-libdvdnav-6.1.1/lib/libdvdnav.dylib -lpthread -liconv /nix/store/cawaxcrrcxx3mipl64760k0p75p3n48q-mujs-1.3.6/lib/libmujs.dylib /nix/store/4frg9qwhvaziyk6m4f1jqmchp204501q-lcms2-2.19.1/lib/liblcms2.dylib /nix/store/92ay55bcwsj41jdk1ng0fvda7cbvd9k7-libarchive-3.8.8-lib/lib/libarchive.dylib /nix/store/zqcg2g5iwlk9n0z6mpqqvrfj3rqs527p-ffmpeg-8.1.2-lib/lib/libavdevice.dylib /nix/store/pp0qn8mxm8mq949wy87km551zrbna0qq-libbluray-1.4.1/lib/libbluray.dylib -lm /nix/store/jfl0bsshv9cg742ig6x90vwn64jak8aq-lua-5.2.4/lib/liblua.dylib /nix/store/4w6nxsq2csf3aj6qggy0rr5fn224ykgx-rubberband-4.0.0/lib/librubberband.dylib -framework Accelerate /nix/store/bnam1gnifvm2s1dab3d23nqfymc8r83c-uchardet-0.0.8/lib/libuchardet.dylib /nix/store/fr71skraxgk85biax0zxrln93zi5bh16-zimg-3.0.6/lib/libzimg.dylib /nix/store/r1rf9w7c2pihgk4snl9xvyr90m9pp3ij-zlib-1.3.2-static/lib/libz.a -framework CoreMedia -framework AVFoundation -framework CoreFoundation -framework CoreAudio -framework AudioUnit -framework AudioToolbox /nix/store/cyxlngr35560g9b4gympq10cs4h6mnp0-openal-soft-1.24.3/lib/libopenal.dylib /nix/store/qxpi14i3ba0sm0m7bb0l8dzizjgwx8fs-libcaca-0.99.beta20/lib/libcaca.dylib -framework OpenGL /nix/store/4xhqv41dryk2i8a98bh5jbqxfnh5n8c5-vulkan-loader-1.4.350.0/lib/libvulkan.dylib -framework CoreVideo
/nix/store/pnd38c1w69ajjfmpsajbz7i72jd2i62v-cctools-binutils-darwin-wrapper-1010.6/bin/ld: line 292: 29878 Trace/BPT trap: 5          /nix/store/zhb8xrwn0mp0xk3argfbh8w73c4cipqs-cctools-binutils-darwin-1010.6/bin/ld "@$responseFile"
clang: error: linker command failed with exit code 133 (use -v to see invocation)
ninja: build stopped: subcommand failed.
```

### Additional context

_No response_

### System metadata

aarch64-darwin

### Notify maintainers

@uninsane @fpletz @SchweGELBin

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.
- [x] I assert that I have read the [automation/AI policy](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#automationai-policy) and that this issue report complies with it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/mp/mpv-unwrapped/package.nix
+++ b/pkgs/by-name/mp/mpv-unwrapped/package.nix
@@ -61,6 +61,7 @@
   wayland-protocols,
   wayland-scanner,
   zimg,
+  llvmPackages,
 
   # Boolean
   alsaSupport ? stdenv.hostPlatform.isLinux,
@@ -163,6 +164,8 @@ stdenv.mkDerivation (finalAttrs: {
     buildPackages.darwin.sigtool
     swift
     makeBinaryWrapper
+    # TODO: Remove once #536365 reaches this branch
+    llvmPackages.lld
   ]
   ++ lib.optionals waylandSupport [ wayland-scanner ];
 
@@ -233,6 +236,9 @@ stdenv.mkDerivation (finalAttrs: {
   # ./osdep/mac/swift.h:270:9: fatal error: '.../app_bridge_objc-1.pch' file not found
   env = lib.optionalAttrs (stdenv.hostPlatform.isDarwin) {
     NIX_SWIFTFLAGS_COMPILE = "-disable-bridging-pch";
+
+    # TODO: Remove once #536365 reaches this branch
+    NIX_CFLAGS_LINK = "-fuse-ld=lld";
   };
 
   postBuild = lib.optionalString stdenv.hostPlatform.isDarwin ''

